documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -121,6 +180,22 @@ AKS support and limitations:
 
 See also the [Azure Kubernetes Service sample]({{<relref "/samples/azure-kubernetes-service/_index.md">}}).
 
+##### Oracle WebLogic Server on the Azure Kubernetes Service Marketplace (WLS on AKS Marketplace)
+
+The WebLogic Server on AKS Marketplace
+offer lets you embrace cloud computing by providing greater choice
+and flexibility for deploying your WLS domains and applications.
+The offer leverages the WebLogic Kubernetes ToolKit to automate
+the provisioning of WebLogic and Azure resources so that you can easily move WLS workloads to AKS.
```

Comment:
WebLogic -> WebLogic Server and Azure resources ...

Reply:
fixed